lib/Doctrine/Import/Builder.php

```diff
@@ -990,6 +1013,75 @@ private function innerBuildActAs($actAs, $level = 0, $parent = null, array &$emi
         return $build;
     }
 
+    /**
+     * Adds the columns of the used actAs behaviors to the comment block.
+     *
+     * @param class-string $className
+     * @param array $instanceOptions
+     *
+     * @throws Doctrine_Import_Builder_Exception
+     */
+    private function determineActAsColumns($className, $instanceOptions)
+    {
+        // No class specified or class does not exist.
+        if (!$className || !class_exists($className)) {
+            return;
+        }
+
+        $actAsInstance = new $className($instanceOptions);
```

Comment:
What if the  constructor can not be invoked? In the case of abstract classes or 'private' constructors, this will throw an error.

Should we handle that case too?

Reply:
I played around with it yesterday, but in PHP < 7, I've not found a clean way to handle it. Exceptions for instantiation problems have been introduced in PHP 7 and I don't think it's worth dealing with reflection here. The worst case would be a crash while generating the model code instead of crashing the code while executing it later. wdyt?

Reply:
I am planning to move this repo to support php >= 7.4 as a first iteration, so we could opt for your proposal to handle exceptions, WDYT?

Reply:
Added the exception handling and, at least for now, an early exit if running below php 7. Just in case this is merged before the min php version change.